README.md

```diff
@@ -37,14 +37,59 @@ install.packages("devtools")
 devtools::install_github("UBC-MDS/labzenr")
 ```
 
-## Example
+## Dependencies
+
+```r
+fs
+stringi
+usethis
+gert
+magrittr
+janitor
+dplyr
+tibble
+tidyr
+checkmate
+credentials
+jsonlite
+stringr
+rlang
+purrr
+```
 
-This is a basic example which shows you how to solve a common problem:
+## Usage
 
-``` r
-library(labzenr)
-## To be written
+In order to show the usage of our package, we provided two dummy labs, one [Rmarkdown lab](https://github.com/UBC-MDS/labzenr/tree/master/inst/extdata/dummylab.Rmd) and one [Jupyter notebook](https://github.com/UBC-MDS/labzenr/tree/master/inst/extdata/dummylab.ipynb).
+
+```r
+>>> library(labzenr)
+# for jupyter notebook:
```

Comment:
to define the file path that works for all systems we can use the following command

notebook <- system.file("extdata", "dummylab.ipynb", package = "labzenr")